es-vue-base/src/lib-icons/icon-video-play.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <svg
+        :style="{
+            height: height,
+            width: width,
+        }"
+        viewBox="0 0 74 54"
```

Comment:
Would the viewBox values need to be changed to be consistent with other base icons or would this icon be in a different category? The video play icon for the EsVideo in es-cms-pages is 74x54px.